README.md

```diff
@@ -4,132 +4,72 @@
-in the event of failure, within predetermined limits. Juju supports backup,
-restore, and rolling upgrade operations appropriate for large-scale
-centralised enterprise grade management and operations systems.
-
-## Get started
-
-Our community hangs out at the [Charmhub discourse](https://discourse.juju.is/)
-which serves as a combination mailing list and web forum. Keep up with the news
-and get a feel for operator engineering and usage there. Get  the Juju CLI on
-Windows, macOS or Linux with the
-[install instructions](https://juju.is/docs/installing) and
-[try the tutorials](https://juju.is/docs/olm/tutorials). All you need is a small
-K8s cluster, or an Ubuntu machine or VM to run MicroK8s.
-
-Read the [documentation](https://juju.is/docs) for a comprehensive reference
-of commands and usage.
-
-## Contributing
-
-Check the [developer instructions](https://juju.is/docs/dev/how-to) to get
-started and follow our [code and contribution guidelines](CONTRIBUTING.md) to
-learn how to make code changes. File bugs in
-[Launchpad](https://bugs.launchpad.net/juju/+filebug) or ask questions on
-our [Freenode IRC channel](https://webchat.freenode.net/#juju), and
-[Mattermost](https://chat.charmhub.io/).
+Juju is an open source application orchestration engine that enables any application operation (deployment, integration, lifecycle management) on any infrastructure (Kubernetes or machines) at any scale (development or production) in the same easy way (typically, one line of code), through special operators called ‘charms’.
+
+
+||||
+|-|-|- |
+| :point_right: | **Learn about Juju!** | Start deploying charms: <br> https://juju.is/docs/juju <br> _Juju is declarative, model-driven, and has native support for integrations and scaling, on any cloud. You want to deploy `foo`? Run `juju deploy foo`! You wish to integrate `foo` and `bar`? Run `juju integrate foo bar`!_ |
+||||
+|| Get better at Juju! | Take advantage of the charm ecosystem: <br> https://charmhub.io/ <br> _Quickly deploy a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a database ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more!_ |
+||||
+|| Make Juju your own! | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
+
+## Use Juju
+Give it a quick try:
+
+_(assumes you have a Linux that supports installation via snap)_
+
+```
+# Use the Multipass 'charm-dev' blueprint to quickly get an Ubuntu VM with a Kubernetes cloud -- MicroK8s -- and Juju: 
+sudo snap install multipass && \
+multipass launch --cpus 4 --memory 8G --disk 30G --name tutorial-vm charm-dev # tune CPU/RAM/HDD accordingly to your needs
```

Comment:
Might be worth marking this as optional? Installing multipass and launching a VM is a pretty hefty first step when all that is really needed is juju itself and microk8s.

```
snap install microk8s --classic
snap install juju
```
might be better? What do you think?

Reply:
True but keeping it all confined in a VM makes it more inviting, as you can easily clean up. Maybe we could just make the VM lighter instead by reducing the cpus, memory, and disk requirements? WDYT?

Reply:
As discussed, I will break this up into 3 code boxes, one with setup, one from `juju bootstrap` onwards, one with `cleanup`.